### DISCUSSION TITLE
Visualize large vector data as rasterized vector tiles

### DISCUSSION THREAD

**Discussion by kaushikCanada** [opengeos/leafmap#196] [Ideas]
One of the things that clients ask is to visualize large datasets. For this we have to do a lot of things like tippecanoe and then tileserver etc. we can have a feature to be able to render large data , convert to COG , create a stac catalog entry , and serve the map as a layer. We have been struggling with gdal command line commands in the absence of such options.

**Comment by giswqs** [comment]
Yes, this would be a nice feature to add. Pull requests are welcome.
